docs/contributing/contributing_test.md

```diff
@@ -149,6 +149,63 @@ Note: If you are implementing a new Expectation, but don’t plan to immediately
 
 You can run just the Expectation tests with `pytest tests/test_definitions/test_expectations.py`.
 
+### Performance testing
+
+##### Configuring Data Before Running Performance Tests
+
+The performance tests use BigQuery.
+
+Before running a performance test, setup data with `tests/performance/setup_bigquery_tables_for_performance_test.sh`.
+
+For example:
+
+```bash
+GE_TEST_BIGQUERY_PEFORMANCE_DATASET=<YOUR_GCP_PROJECT> tests/performance/setup_bigquery_tables_for_performance_test.sh
+```
+
+For more information on getting started with BigQuery, please refer to the [above section on BigQuery tests](#bigquery-tests).
+
+##### Running the Performance Tests
+
+Run the performance tests with pytest, e.g.
+
+```
+pytest tests/performance/test_bigquery_benchmarks.py \
+  --bigquery --performance-tests \
+  -k test_bikeshare_trips_benchmark[1] \
```

Comment:
for this flag, pytest actually required this for me, with the single quotes and `True`

```bash
  -k 'test_bikeshare_trips_benchmark[1-True]' 
```